tests/unit/hubble/test_hubio.py

```diff
@@ -212,6 +223,41 @@ def _mock_post(url, data, headers=None, stream=True):
     )
 
 
+def test_push_with_authorization(mocker, monkeypatch, tmpdir):
+    clear_function_caches()
+    os.environ['JINA_HUB_ROOT'] = str(tmpdir)
```

Comment:
u should use a fixture to make sure that this env variable is properly set and unset at the beginning and at the end of the test regardless of the success of failure of it

Reply:
I created `auth_token` fixture. Please check it out.